addons/controllers/kappcontrollerconfig_controller_test.go

```diff
@@ -9,11 +9,12 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
```

Comment:
this new line is auto generated by `make fmt` from root make target, that make linter happy